core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowAkkaPlugin.scala

```diff
@@ -25,10 +25,29 @@ import spray.json._
 import cloudflow.sbt.CloudflowKeys._
 import cloudflow.blueprint.StreamletDescriptorFormat._
 import CloudflowBasePlugin._
+import com.lightbend.sbt.javaagent.JavaAgent
+import com.lightbend.sbt.javaagent.JavaAgent.JavaAgentKeys.resolvedJavaAgents
+import java.io.File
 
 object CloudflowAkkaPlugin extends AutoPlugin {
+  val AppRunner: String = "akka-entrypoint.sh"
+  val AppHome           = "${app_home}"
```

Comment:
How is  this used?

Reply:
`akka-entrypoint.sh` uses `$app_home` - hence it needs to be set which is done here. This is what was earlier as well. But I guess this will change when we have a separate base image for akka.

Reply:
What sets this `$app_home` variable in the `akka-entrypoint.sh` script? (just curious)